docs/user_guide/3_getting_started.md

```diff
@@ -178,11 +178,194 @@ When running the one-liner, along with the icon that starts the Grasper, you wil
+* **Program Motors:** Now you can click the ```Bootload Motors``` button. The process is fairly slow, and takes about a 30 second per motor.
+
+### Change controllers
+Use the *Change Controllers* plugin to load one of the three different types of controllers set by default. Simply click on a controller type, and it will call a service from the pr2_controller_manager to unload the currently running controller if necessary, and load the one you've selected. See the chapter on control for details of these algorithms.
+	**Plugins → Shadow Robot → Change Controllers**
+
+![selecting_different_control_mode](../img/selecting_different_control_mode.png)
+
+### Advanced controllers
+Apart from the three standard controls, you can set the parameters for different control strategies (host – motor) from this plugin.
+	**Plugins → Shadow Robot → Advanced → Advanced Controls**
+
+  ![selecting_different_control_strategies](../img/selecting_different_control_strategy.png)
+
+  **NOTE: CURRENTLY THE ONLY FULLY SUPPORTED TYPES ARE POSITION - PWM CONTROL** *(position control),* **AND EFFORT - TORQUE CONTROL** *(teach mode control).* **SELECTING OTHER TYPES MAY CAUSE UNPREDICTABLE RESULTS AND DAMAGE THE HARDWARE.**
+
+  ### Motor Resetter
+If for some reason you need to reset the firmware on a motor, you can either press the reset button on the PCB itself (which requires removal of the base covers), or use this plugin.
+	**Plugins → Shadow Robot → Basic → Motor Resetter**
+
+  ![resetting_motor_microcontrollers](../img/resetting_motor_microcontrollers.png)
+
+  Tick the motors you wish to reset, and click ```Reset Motors```. You should see the corresponding joints jiggle as the motors auto-zero the strain gauges.
+
+### Joint Sliders
+A simple interface has been provided to control the position of each joint using a slider (you have to start the position control first).
+	**Plugins → Shadow Robot → Joint Sliders**
+    ![joint_sliders](../img/joint_sliders.png)
+
+
+  A window with twenty sliders will appear. Moving any slider will cause the corresponding joint on the hand to move.
+
+### Hand Calibration
+This plugin is used internally by Shadow to calibrate the raw data from the position sensors.
+	**Plugins → Shadow Robot → Basic → Shadow Hand Calibration**
+
+  ![calibrating_joint_sensors](../img/calibrating_joint_sensors.png)
+
+  It’s very unlikely that the sensors moved inside of the hand, BUT, if you find misalligments with the model and you require a re-calibration, contact Shadow Robot Company here: <support@shadowrobot.es>.
 
-## Graphical User interface
 
 ## Command line interface
-### Using rostopic to view sensors
-### Using rostopic to view debugging data
-### Reset motors
+All functions of the hand are available from the command line.
+### Using rostopic
+
+You can find all the information about the topic published in the Shadow Hand from this link:
+
+<https://shadowrobot.atlassian.net/wiki/spaces/HANDEG/pages/63569986/Hand+E+ROS+Kinetic+Topics>
```

Comment:
We need to remove this link and add the topics here.

Reply:
No problem. There's quite a lot of information on that topics page, would you like me to add all of it or just a particular subset?